packages/extension-mcp/src/utils.ts

```
@@ -17,8 +17,9 @@
     EmbeddedResource,
     ReadResourceResult
 } from '@modelcontextprotocol/sdk/types.js'
-import { Context } from 'koishi'
+import { Context, Logger } from 'koishi'
```

ESLint

[error] 'Logger' is defined but never used.

(@typescript-eslint/no-unused-vars)